app/utils.js

```diff
@@ -6,4 +6,11 @@ export const jsonStringify = (json) => {
         }
         return value;
     })
+}
```

Comment:
I guess you have copies this code from:
https://stackoverflow.com/questions/105034/create-guid-uuid-in-javascript

Notice that the author says it is not the best solution and node-uuid is better.
I would not change it now but just keep that in mind

Reply:
Done, i replaced it with node-uuid, i think we will need it in the future